### PR TITLE
Fix path of smart_proxy_salt (s)/bin scripts

### PR DESCRIPTION
**Pull request by sbernhard**

### CODE REVIEW

**Comment by sbernhard** [comment]
@ekohl what do you think about this? As said, other smart proxys are affected, too. Can you confirm this (e.g. for omaha, openscap)?

File: packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec [hunk 1]
```diff
@@ -176,16 +178,18 @@ EOF
...
 %exclude %{gem_instdir}/cron
 
-
 %files doc
 %doc %{gem_docdir}
 %doc %{gem_instdir}/README.md
 
 %changelog
+* Thu Jul 09 2020 Bernhard Suttner <suttner@atix.de> - 3.1.2-6
+- Fix upload-salt-report pat
```
Comment (ekohl) [inline packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec:190]:
```suggestion
- Fix upload-salt-report path
```

**Comment by sbernhard** [comment]
Have a look at /usr/bin/smart_proxy_openscap_send (https://yum.theforeman.org/plugins/2.1/el7/x86_64/tfm-rubygem-smart_proxy_openscap-0.7.2-6.fm2_1.el7.noarch.rpm) which uses "#!/usr/bin/env ruby". I guess this can not work because of wrong ruby environment.

**Comment by ekohl** [comment]
You're right, that can't work. I think the delivery team should take a look.

**Comment by sbernhard** [comment]
> I wonder why this should be in sbin.

upload-salt-reports loads /etc/salt/master which is only readable for root. Is this a reason to put the upload script in /sbin?

**Comment by ekohl** [comment]
Hmm, fair point. I sort assumed we had fixed shebangs in the template. Looks like we don't:
https://github.com/theforeman/foreman-packaging/blob/8243ef3d8510d743321cb8321ad8a6c282dfd8ed/gem2rpm/smart_proxy_plugin.spec.erb#L124-L130

**Comment by sbernhard** [comment]
Anything I need to change before this can be merged?

**Comment by evgeni** [comment]
Nope, just time for someone to review this :)

**Comment by evgeni** [comment]
@sbernhard will you take care of opening any cherry picks against stable branches that might be needed?

**Pull request merged**